apps/settings/serializers/public.py

```diff
@@ -34,6 +34,8 @@ class PrivateSettingSerializer(PublicSettingSerializer):
     PASSWORD_RULE = serializers.DictField()
     SECURITY_SESSION_SHARE = serializers.BooleanField()
     XPACK_LICENSE_IS_VALID = serializers.BooleanField()
+    XPACK_LICENSE_EDITION_ULTIMATE = serializers.BooleanField()
+    FACE_RECOGNITION_ENABLED = serializers.BooleanField()
     XPACK_LICENSE_INFO = serializers.DictField()
     HELP_DOCUMENT_URL = serializers.CharField()
     HELP_SUPPORT_URL = serializers.CharField()
```

Comment:
There are no obvious discrepancies in the given code provided, so I can't identify any issues, irregularities to be addressed, or suggestions on optimization. The codes appear correct and up-to-date according to the specified knowledge cutoff (2021-09-01) and current date (2024-12-19).